CHANGELOG.rst

```diff
@@ -19,11 +19,13 @@ User-Facing Changes
 - **UI**: Add a playful design for the 404 and application error pages (`#3248 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3248>`__).
 - **UI**: Update redirect page styles (`#3257 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3257>`__).
 - **UI**: Remove Renku 2.0 beta warning alert from dashboard (`#3357 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3357>`__).
+- **UI**: Fix permission checks in Renku 2.0 (`#3351 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3351>`__).
```

Comment:
This should be under user facing bug fixes:
```suggestion
- **UI**: Fix bug in Renku 2.0 where group members could only have permissions on projects if they were also direct project members (`#3351 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3351>`__).
```

Reply:
Users have permissions, but the UI did not reflect that.

Reply:
Done in b7da33d8bc0670204f9320715793a9a990c59b6c.